Automattic/woocommerce-payments
# Fix/8175 show tooltip instead of deposit guesstimate

Fixes #8175 

<img width="962" alt="Screenshot 2024-02-14 at 12 42 39 PM" src="https://github.com/Automattic/woocommerce-payments/assets/4167300/d3b73a82-7745-4fe5-9293-7c79e8a24374">

Tooltip text is as follows:

> Deposits are initiated based on the following criteria:
> - The [pending period](https://woo.com/document/woopayments/deposits/deposit-schedule/#pending-period-chart) in your country
> - Your account's [available funds](https://woo.com/document/woopayments/deposits/deposit-schedule/#available-funds)
> - Your [deposit schedule](https://woo.com/document/woopayments/deposits/change-deposit-schedule/) settings

#### Changes proposed in this Pull Request
- Removes the estimated next deposit date from deposit overview screen (and related code)
- Adds a tooltip with more info and links to docs about what factors affect deposits / payouts

#### Testing instructions

* Get your store in a state where you can expect payouts (e.g. outside of new account waiting period)
* View `Payments > Overview`
* Check the deposit box payout schedule info makes sense, and the tooltip is helpful.
* Test different payout schedules – the format string(s) for the schedule summary changed in this PR (to remove the estimated date part).

Bonus points:

- Test other store states to ensure the copy & tooltip are relevant and helpful. (Should be consistent/unchanged from previous logic)

-------------------


**Post merge**


- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : qa testing not required
- [x] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable. no changes to critical flows
- [x] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable. TBD - will need adding to What's new post, with link to relevant P2 discussion

## more screenshots

With a monthly schedule:

<img width="905" alt="Screenshot 2024-02-15 at 9 54 26 AM" src="https://github.com/Automattic/woocommerce-payments/assets/4167300/c0b73f74-1130-43ce-9b8b-327730726822">
